http4s/src/main/scala/com.snowplowanalytics.snowplow.collector.core/HttpServer.scala

```diff
@@ -33,20 +32,24 @@ object HttpServer {
     app: HttpApp[F],
     interface: String,
     port: Int,
-    secure: Boolean
+    secure: Boolean,
+    maxConnections: Int,
+    idleTimeout: FiniteDuration
```

Comment:
What about passing `networking` ? Building the HTTP server probably will likely always require all the parameters of `Networking`

Reply:
netty does not require maxConnections, but I guess it can very well be `networking` instead of the two separate params 👍